<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>609</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="widget_exp2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>751</width>
         <height>601</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_exp2_function" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_exp2_quickPing" native="true">
             <property name="styleSheet">
              <string notr="true">QWidget#widget_exp2_quickPing{
	background-color: rgb(211, 211, 206);
border: 1px solid #747d8c; 
border-radius: 5px;}</string>
             </property>
             <widget class="QWidget" name="widget_exp2_pingIP" native="true">
              <property name="geometry">
               <rect>
                <x>11</x>
                <y>33</y>
                <width>701</width>
                <height>33</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="spacing">
                <number>5</number>
               </property>
               <property name="leftMargin">
                <number>5</number>
               </property>
               <property name="topMargin">
                <number>5</number>
               </property>
               <property name="rightMargin">
                <number>5</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; vertical-align:sub;&quot;&gt;IP地址&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; vertical-align:sub;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_exp2_pingIP1">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>39</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_exp2_pingIP2">
                 <property name="text">
                  <string>156</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_exp2_pingIP3">
                 <property name="text">
                  <string>66</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; vertical-align:sub;&quot;&gt;从&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_exp2_pingIP4">
                 <property name="text">
                  <string>5</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; vertical-align:sub;&quot;&gt;到&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_exp2_pingIP5">
                 <property name="text">
                  <string>30</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; vertical-align:sub;&quot;&gt;超时&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_exp2_pingTime">
                 <property name="text">
                  <string>1000</string>
                 </property>
                 <property name="maxLength">
                  <number>8</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>ms</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; vertical-align:sub;&quot;&gt;线程：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_exp2_pingCount">
                 <property name="maximumSize">
                  <size>
                   <width>43</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>20</string>
                 </property>
                 <property name="maxLength">
                  <number>4</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QTableWidget" name="tableWidget_exp2_pingResult">
              <property name="geometry">
               <rect>
                <x>11</x>
                <y>73</y>
                <width>711</width>
                <height>471</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">selection-color: rgb(214, 211, 204);</string>
              </property>
              <column>
               <property name="text">
                <string>IP地址</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>网卡地址</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>主机名</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>返回信息</string>
               </property>
              </column>
             </widget>
             <widget class="QPushButton" name="pushButton_exp2_pingStart">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>550</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>开始</string>
              </property>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>-5</y>
                <width>731</width>
                <height>31</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(75, 98, 152);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(8, 35, 105), stop:1 rgba(168, 201, 238));
color: rgb(255, 255, 255);
</string>
              </property>
              <property name="text">
               <string> QuickPing</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
